apps/services/auth/admin-api/src/main.ts

```diff
@@ -14,4 +15,15 @@ bootstrap({
   healthCheck: {
     database: true,
   },
+  beforeServerStart: async (app) => {
```

Comment:
The Coderabbit comment hinted me to check NestJS Docs for middlewares. Turns out they have nice support for middleware

https://docs.nestjs.com/middleware#functional-middleware

And we can use `forRoute` to only apply this middleware for the Zendesk webhook route, that way we are only applying this to the middleware chain for that route which require this.